.github/workflows/embedding-sdk.yml

```diff
@@ -186,3 +186,10 @@ jobs:
               }
             });
             process.exit(1);
+
+  cross-version-for-breaking-changes:
```

Comment:
we should put this above the `embedding-sdk-tests-result` job, which should always be last - but definitely not make this a `needs:` of that job (which makes it required)